src/ensembl/src/content/app/browser/browser-nav/BrowserNavBar.scss

```diff
@@ -12,4 +14,49 @@
   &.browserNavBarExpanded {
     width: 100vw;
   }
+
+  dd {
+    margin-left: 10px;
+  }
+
+  form {
+    display: inline-block;
+  }
+
+  label {
+    font-weight: 300;
+    margin: 0 5px 0 10px;
+  }
+
+  input {
+    padding: 3px 9px;
+  }
+}
+
+.errorText {
+  color: $red;
+}
+
+.inputText {
```

Comment:
this class is used only in the BrowserRegionField component. Why doesn't it sit in the scss file there?